app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -30,6 +32,11 @@ class Uploader extends \Magento\Backend\Block\Widget
      */
     protected $_fileSizeService;
 
+    /**
+    * @var \Magento\Framework\Image\Adapter\ConfigInterface
+    */
+    protected $imageConfig;
```

Comment:
Due to [Magento backward-compatible guide](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/) we can't add protected properties. All new properties should be private.